lib/audit/checksum.rb

```diff
@@ -8,22 +8,31 @@ class Checksum
   # rubocop:disable Style/EmptyMethod:
   # TODO: implement this;  we begin with a placeholder
 
-  def self.validate_disk(last_checked_b4, endpoint, algorithm="MD5")
+  def self.validate_disk(last_checked_b4, storage_dir)
```

Comment:
so we have a ticket to wire this method up to the actual code when it's ready, yes??  We could put in the timestamp code and so on analogous to M2C / C2M checks now??  (or put that info in ticket?)

Reply:
i'll go ahead a ticket it, #595